docs/source/AdministratorGuide/ServerInstallations/InstallingWebAppDIRAC.rst

```diff
@@ -256,154 +261,219 @@ Install and configure NGINX
 Note: you can run NGINX in a separate machine.
 
 The official site of NGINX is the following: `<http://nginx.org/>`_
-The required NGINX version has to be grater than 1.4.
+The required NGINX version has to be grater than 1.4 and WebDAV nginx module to serve static files.
+
+* Prepare, needed the development repository to compile the WebDAV dynamic module for Nginx::
 
-  * Install Nginx using package manager::
+      yum update -y
+      yum groupinstall "Development Tools" -y
+      yum install yum-utils pcre-devel zlib-devel libxslt-devel libxml2-devel -y
 
-         yum install nginx
+* Install Nginx using package manager. At this point, you should be able to install the pre-built Nginx package with dynamic module support::
 
+      yum install nginx -y
+      systemctl enable nginx
+      systemctl start nginx
 
 If your version is not grater than 1.4 you have to install NGINX manually.
 
 * Manual install::
 
-     vim /etc/yum.repos.d/nginx.repo
+      vim /etc/yum.repos.d/nginx.repo
 
 CentOS::
 
-      [nginx]
-      name=nginx repo
+      [nginx-stable]
+      name=nginx stable repo
       baseurl=http://nginx.org/packages/centos/$releasever/$basearch/
-      gpgcheck=0
+      gpgcheck=1
       enabled=1
+      gpgkey=https://nginx.org/keys/nginx_signing.key
+      module_hotfixes=true
+
+      [nginx-mainline]
+      name=nginx mainline repo
+      baseurl=http://nginx.org/packages/mainline/centos/$releasever/$basearch/
+      gpgcheck=1
+      enabled=0
+      gpgkey=https://nginx.org/keys/nginx_signing.key
+      module_hotfixes=true
 
 RHEL::
 
-      [nginx]
-      name=nginx repo
+      [nginx-stable]
+      name=nginx stable repo
       baseurl=http://nginx.org/packages/rhel/$releasever/$basearch/
-      gpgcheck=0
+      gpgcheck=1
       enabled=1
+      gpgkey=https://nginx.org/keys/nginx_signing.key
+      module_hotfixes=true
+
+      [nginx-mainline]
+      name=nginx mainline repo
+      baseurl=http://nginx.org/packages/mainline/rhel/$releasever/$basearch/
+      gpgcheck=1
+      enabled=0
+      gpgkey=https://nginx.org/keys/nginx_signing.key
+      module_hotfixes=true
 
 Due to differences between how CentOS, RHEL, and Scientific Linux populate the $releasever variable, it is necessary to manually replace $releasever with either 5 (for 5.x) or 6 (for 6.x),
 depending upon your OS version. For example::
 
-   [nginx]
-   name=nginx repo
-   baseurl=http://nginx.org/packages/rhel/6/$basearch/
-   gpgcheck=0
-   enabled=1
+      [nginx]
+      ...
+      baseurl=http://nginx.org/packages/rhel/6/$basearch/
+      ...
```

Comment:
The instructions above are for SL5 and SL6. I would remove SL5, which is not supported anymore, add CC7, and maybe even remove SL6?

Reply:
fixed